### PR TITLE
Fix running nG Python tests on Windows and update doc

### PR DESCRIPTION
**Pull request by mbencer**

### CODE REVIEW

**Review by jiwaszki — APPROVED** [review]

File: ngraph/python/tests/test_ngraph/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+if sys.platform == "win32":
+    # ngraph.dll is installed in PYTHON_INSTALL_DIR/Lib as default
+    # (which is 2 directories above ngraph module)
+    # and this path needs to be visible to the _pyngraph module.
+    #
+    #
+    # If you're using a custom installation of nGraph,
+    # add the location of ngraph.dll to your system PATH.
+    ngraph_dll_dir = os.path.join(os.path.dirname(ng.__file__), ".", ".")
```
Comment (tomdol) [inline ngraph/python/tests/test_ngraph/__init__.py:13]: Shouldn't it be `..` twice instead of single dots?

Comment (mbencer) [reply ngraph/python/tests/test_ngraph/__init__.py:13]: interesting, single dot works on win too, but you are right ".." is more clear

File: ngraph/python/BUILDING.md [hunk 2]
```diff
@@ -36,15 +36,23 @@ cmake ..
...
       -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE
-      -DCMAKE_CXX_COMPILER=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.16.27023\bin\Hostx64\x64
+      -DCMAKE_CXX_COMPILER="C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.16.27023\bin\Hostx64\x64"
 ~~~~
 There are a couple of things to notice here. One is that the full path to the x64 version of
 MSVC compiler has to be specified. This is because DNNL requires a 64-bit version and cmake may
 fail to detect it correctly.
 The other equally important thing to note is that the temporary directory where the build is to be installed can be specified.
-This examples uses `C:\temporary_install_dir` however, a subdirectory of `ngraph\build` works as well.
+If the installation directory is not specified, the default location is `C:\Program Files\OpenVPN`.
```
[the remark this answers is not in the document]
Comment (mbencer) [reply ngraph/python/BUILDING.md:45]: right ;) good catch

File: ngraph/python/tests/test_ngraph/__init__.py (hunk 1 above)
Comment (jiwaszki) [reply ngraph/python/tests/test_ngraph/__init__.py]: @mbencer Are you sure "." were working? It shouldn't be possible in any case.

Comment (tomdol) [reply ngraph/python/tests/test_ngraph/__init__.py]: I think the `import ngraph as ng` statement in this file does the trick. It imports the whole ngraph module, and the `__init__.py` file for that module does `from ngraph.impl import Node`. The `ngraph.impl` module sets the correct paths.

I think we can even remove the whole if statement from `test_ngraph/__init__.py` and it should still work.

Comment (mbencer) [reply ngraph/python/tests/test_ngraph/__init__.py]: @jiwaszki you are right, as @tomdol mentioned alone `import ngraph` causes that ngraph.dll dir path is added to `PATH` so it is the reason that is works for me (single dot is not correct). Content of if statement is not needed.

File: ngraph/python/BUILDING.md [hunk 3]
```diff
@@ -56,7 +64,13 @@ adjust the number of threads used in the building process to your machine's capa
...
     `cmake --build . --target python_wheel --config Release -j 8`
 
-The final wheel should be located in `build\python\dist` directory.
+The final wheel should be located in `ngraph\python\dist` directory.
+
+4. Configure the environment for the Inference Engine Python API:
+
+    `call <CMAKE_INSTALL_PREFIX>\bin\setupvars.bat`
+
+NOTE: This step is not required for using alone nGraph Wheel, but it needed for most usage scenarios (for example for run unit tests).
```
Comment (aalborov) [inline ngraph/python/BUILDING.md:73]: Could you please change the note to the following?
```
> **NOTE**: Skip this step if you want to use the nGraph Wheel by itself. This step is required for most usage scenarios, like running unit tests.
```

**Review by aalborov — CHANGES_REQUESTED** [review]

**Review by tomdol — APPROVED** [review]

File: ngraph/python/BUILDING.md (hunk 3 above)
Comment (mbencer) [reply ngraph/python/BUILDING.md:73]: done

**Review by postrational — APPROVED** [review]

File: ngraph/python/BUILDING.md [hunk 4]
```diff
@@ -81,6 +95,6 @@ Unit tests require additional packages be installed:
 
 Then run tests:
 
-    (venv) $ pytest test/ngraph/
+    (venv) $ pytest tests
 
 [ngraph_build]: http://ngraph.nervanasys.com/docs/latest/buildlb.html
```
Comment (lazarevevgeny) [inline ngraph/python/BUILDING.md:100]: Should we remove reference to nGraph in http://ngraph.nervanasys.com?

**Review by aalborov — APPROVED** [review]

**Pull request merged**